README.md

```diff
@@ -0,0 +1,39 @@
+# elastic-meetup-demo
```

Comment:
change to: Elastic Meetup Demo

A text is missing which is explaining that this is part of the demo which took place at the 3d Elastic Athens Meetup (https://www.meetup.com/Greece-Elastic/events/259473881/). Furthermore, the presentation can be found here: https://www.slideshare.net/StamatisKatsaounis/monitoring-the-nats-messaging-system-at-scale-with-elastic-beats